jsapi/src/main.js

```diff
@@ -6,28 +6,75 @@ goog.require('ol.events');
 goog.require('ol.Map');
 goog.require('ol.Overlay');
 goog.require('ol.View');
+goog.require('ol.layer.Image');
 goog.require('ol.layer.Tile');
 goog.require('ol.source.OSM');
+goog.require('ol.source.ImageWMS');
+goog.require('ol.source.WMTSRequestEncoding');
 
 /**
  * @classdesc
  * The map is the core component of the Geoportail V3 API.
  *
  * @constructor
  * @extends {ol.Map}
- * @param {olx.MapOptions} options Map options.
+ * @param {luxx.MapOptions} options Map options.
  * @export
  */
 lux.Map = function(options) {
-  options.view = new ol.View({
-    zoom: 5,
-    center: [0, 0]
-  });
-  options.layers = [new ol.layer.Tile({
-    source: new ol.source.OSM()
-  })];
+
+  var layers    = [];
+  var defaultBg = 'basemap_2015_global';
+
+  /**
+   * @private
+   * @type {Object} config A JSON layers config object.
+   */
+  this.layersConfig = null;
+
+  this.promise = null;
+
+  if (options.bgLayer) {
+    layers.push(options.bgLayer);
+    delete options.bgLayer;
+  } else {
+    layers.push(defaultBg);
+  }
+  if (options.layers) {
+    layers = layers.concat(options.layers);
+    delete options.layers;
+  }
+  this.promise = fetch('../layers.json').then(function(resp) {
+    return resp.json();
+  }).then(function(json) {
+    this.layersConfig = /** @type {luxx.LayersOptions} */ (json);
+    this.addLayers_(layers);
+  }.bind(this));
+
+  var viewOptions = {
+    center : ol.proj.fromLonLat([6.215, 49.845]),
```

Comment:
What's the purpose of `fromLonLat` here? Didn't you forget to convert this to `3857`?